testdata/ansible/memcached-operator/requirements.yml

```
@@ -2,4 +2,5 @@
 collections:
   - name: community.kubernetes
     version: "==1.1.1"
-  - operator_sdk.util
+  - name: operator_sdk.util
+	  version: "0.1.0"
```

Needs to be regenerated
```suggestion
    version: "0.1.0"
```